tests/publiccloud/aws_cli.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in EC2 using aws binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use registration;
+use warnings;
+use testapi;
+use mmapi;
+use strict;
+use utils;
+use publiccloud::utils "select_host_console";
+
+sub run {
+    my ($self, $args) = @_;
+    $self->select_serial_terminal;
+    my $job_id = get_current_job_id();
+
+    add_suseconnect_product 'sle-module-public-cloud';
+    zypper_call 'in aws-cli jq python3-susepubliccloudinfo';
+
+    set_var 'PUBLIC_CLOUD_PROVIDER', 'EC2';
+    my $provider = $self->provider_factory();
+
+    my $image_id = script_output("aws ec2 describe-images --filters 'Name=name,Values=suse-sles-15-sp3-v*-x86_64' 'Name=state,Values=available'  --output=json | jq -r '.Images[] | select( (.Name | contains(\"-ecs\") | not)).ImageId' | head -n1", 240);
+    record_info("PINT", "Pint query: " . $image_id);
```

Comment:
I am bit lost here , you doing "describe-images" but called it - "pint query" . yes from pint you can get something similar but you not using pint here